src/main/java/com/github/splendor_mobile_game/game/model/Game.java

```diff
@@ -16,16 +18,24 @@ public class Game {
     private TokenList onyxTokens;
     private TokenList goldTokens;
 
-    private ArrayList<Card> cardTier1List = new ArrayList<>();
-    private ArrayList<Card> cardTier2List = new ArrayList<>();
-    private ArrayList<Card> cardTier3List = new ArrayList<>();
+    private Map<CardTier,Deck> revealedDecks = new HashMap<CardTier,Deck>();
+    private Map<CardTier,Deck> decks = new HashMap<CardTier,Deck>();
 
     private int maxTokenStack = 7; // Default number of each token type
 
     private Database database;
 
     public Game(Database database) {
         this.database = database;
+        database.loadCards();
+
+        decks.put(CardTier.LEVEL_1,new Deck(CardTier.LEVEL_1));
+        decks.put(CardTier.LEVEL_2,new Deck(CardTier.LEVEL_2));
+        decks.put(CardTier.LEVEL_3,new Deck(CardTier.LEVEL_3));
+
+        revealedDecks.put(CardTier.LEVEL_1,new Deck(CardTier.LEVEL_1));
+        revealedDecks.put(CardTier.LEVEL_2,new Deck(CardTier.LEVEL_2));
+        revealedDecks.put(CardTier.LEVEL_3,new Deck(CardTier.LEVEL_3));
```

Comment:
I don't understand why would you put into `decks` and `revealedDecsk`, a new empty Deck with certain size. Their size is ALWAYS the same. It is static. Why is it in constructor?????

Reply:
Could you elaborate a bit? I don't understand what you mean. :c

Reply:
Creating a Deck object with empty ArrayList is redundant (since Deck is extending ArrayList, we should provide an ArrayList while we create a Deck object). This code can be simplified to something like decks = new Deck(database.getTier1Cards(), ....).